solver/src/liquidity_collector.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
 
 pub struct LiquidityCollector {
     pub uniswap_liquidity: UniswapLikeLiquidity,
+    pub sushiswap_liquidity: UniswapLikeLiquidity,
```

Comment:
just thinking out loud that we could also have a list of `UniswapLikeLiquidity` objects (in case we don't e.g. create SushiSwap liquidity for the e2e test or it isn't available on a certain network). This would also allow us to configure the UniswapLikeLiquidity which is something I'd prefer (in case we run into issues with the Uni/Sushi mix and we want to turn off one source). But this could also be achieved with an optional field.

Reply:
Right. I had included this as an option on the PR description. Definitely planning to change this.